website/docs/reference/resource-configs/alias.md

```diff
@@ -4,10 +4,47 @@ description: "Read this guide to understand the alias configuration in dbt."
 datatype: string
 ---
 
-:::caution Heads up!
-This is a work in progress document. While this configuration applies to multiple resource types, the documentation has only been written for seeds.
 
-:::
+<Tabs>
+<TabItem value="model" label="Model">
+
+Specify a custom alias for a model in your `dbt_project.yml` file. 
+
+For example, if you have a model that calculates `sales_metric`s and want to give it a more user-friendly alias, you can alias it like this:
```

Comment:
Metrics might be a confusing term to give here given the other sort of metrics dbt supports

Reply:
changed it to `sales_total`

```
models:
  your_project:
    sales_total:
      +alias: sales_dashboard
```